rllib/env/wrappers/pettingzoo_env.py

```diff
@@ -70,6 +70,8 @@ def __init__(self, env):
         super().__init__()
         self.env = env
         env.reset()
+        self._skip_env_checking = True  # TODO avnishn - remove this after making
+        # petting zoo env compatible with check_env
```

Comment:
nit nit, multiline comment usually is above the statement. this format looks a bit weird.